build/build.sh

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+#!/bin/bash
```

Comment:
I always see `#!/bin/bash` before the copyrights, but not sure if it's required to be at the first line.  Just thinking out loud, I don't think it breaks anything.